cordova-lib/src/plugman/fetch.js

```diff
@@ -147,6 +146,9 @@ function fetchPlugin(plugin_src, plugins_dir, options) {
         checkID(options.expected_id, result.pinfo);
         metadata.save_fetch_metadata(plugins_dir, result.pinfo.id, { source: result.fetchJsonSource });
         return result.dest;
+    })            
+    .fail(function(error) {
```

Comment:
I'm not 100% sure, but I think I needed to add this during one of my iterations. It very well could be no-op now. I will see if I can hit it and if it is needed.